Dockerfiles/Dockerfile.frontend

```diff
@@ -1,8 +1,14 @@
-FROM node:20-bookworm
+FROM node:20-bookworm as build
 
 ENV REACT_APP_PROXY=http://localhost:9081
 ENV REACT_APP_NOMOCK=on
 WORKDIR /web/frontend
 COPY ../web/frontend .
-RUN npm install
+RUN npm ci
+RUN npm run build
+
+FROM node:20-bookworm as app
+
+COPY --from=build /web/frontend/ .
+
 ENTRYPOINT ["npm", "start"]
```

Comment:
Does that work? You don't need `nodejs` anymore, as the `npm run build` will create just a set of static html/js/css files. So you can do:

https://github.com/c4dt/service-stainless/blob/main/webapp/Dockerfile#L11

Or 

https://github.com/c4dt/service-stainless/commit/bfeac27eb49bbd154246ff848c60c6400414390f#diff-06839e9bf05123a2ab7c9a5d721be661be9d231a6337bc7b8b0b6d8d8bf53d44R11

Or any other lightweight web server image.

Reply:
I was testing it and it initially seemed fine, but now it's not working, so I put it back into Draft

before that it sort of complained that there was no package.json file as npm ci does not seem to generate one, so I copied everything over to test that

thanks for the comments, I think the error is in there somewhere (I was looking at the Dockerfiles just now, I think https://github.com/c4dt/service-spindle/blob/main/Dockerfile is probably the closest)

Reply:
> I was looking at the Dockerfiles

The spindle Dockerfile uses the `node` image, which is not necessary, as you should only have static files to serve for the frontend. So either a python or a webserver for static content is enough. Here is another, even more minimal webserver:

https://lipanski.com/posts/smallest-docker-image-static-website